pyscf/gto/basis/vdzp.dat

```diff
@@ -0,0 +1,3442 @@
+# Generated via Basis Set Exchange CLI on Feb 17 2024
```

Comment:
It seems that the basis set will be included in BSE.

Reply:
TBD - unsure what the timeline on that will be, see https://github.com/MolSSI-BSE/basis_set_exchange/issues/266. I used the ``bse convert-basis`` command-line utility ([ref](https://molssi-bse.github.io/basis_set_exchange/conversion.html)) to convert the Turbomole-formatted basis set into NWChem format, which is why this note is there.

Reply:
@corinwagen: Thanks to @susilehtola and @bennybp, the full vDZP basis set for ωB97X-3c (https://pubs.aip.org/aip/jcp/article/158/1/014103/2867476/B97X-3c-A-composite-range-separated-hybrid-DFT) has been merged in https://github.com/MolSSI-BSE/basis_set_exchange/pull/316 into the basis set exchange data base. This should (in the long term) also resolve this issue.